eq/deflect/eventHandler.cpp

```diff
@@ -36,6 +36,56 @@ namespace
 {
 typedef std::vector< EventHandler* > EventHandlers;
 static lunchbox::PerThread< EventHandlers > _eventHandlers;
+
+// Values come from QtCore/qnamespace.h, but don't want to depend on Qt just for that
```

Comment:
Could be an option as well. Deflect only states that 'key' contains values from QKeyEvent::key(), so it's up to client code to resolve it. The symbolic name list for all available special keys is ~50 though...
@rdumusc Opinion on this?